Configuration/Base.xcconfig

```diff
@@ -66,17 +66,12 @@ MACOSX_DEPLOYMENT_TARGET_1400 = 10.13;
 MACOSX_DEPLOYMENT_TARGET_1500 = 10.14;
 MACOSX_DEPLOYMENT_TARGET = $(MACOSX_DEPLOYMENT_TARGET_$(XCODE_VERSION_MAJOR));
 WATCHOS_DEPLOYMENT_TARGET_1400 = 4.0;
-WATCHOS_DEPLOYMENT_TARGET_1500 = 5.0;
+WATCHOS_DEPLOYMENT_TARGET_1500 = 4.0;
 WATCHOS_DEPLOYMENT_TARGET = $(WATCHOS_DEPLOYMENT_TARGET_$(XCODE_VERSION_MAJOR));
 TVOS_DEPLOYMENT_TARGET_1400 = 11.0;
 TVOS_DEPLOYMENT_TARGET_1500 = 12.0;
 TVOS_DEPLOYMENT_TARGET = $(TVOS_DEPLOYMENT_TARGET_$(XCODE_VERSION_MAJOR));
 
-REALM_LD_CLASSIC_1400 = ;
-REALM_LD_CLASSIC_1500 = -Wl,-ld_classic;
```

Comment:
This was specifically a workaround for a Xcode 15.0 bug that was fixed in 15.1 (and is why this is bumping to 15.1 minimum rather than 15.0).